app/assets/stylesheets/module-admin.scss

```diff
@@ -412,3 +412,141 @@ trix-toolbar .button_group button.icon, trix-toolbar .button_group input[type=bu
 	width: 2em;
 	font-size: 14px;
 }
+
+
+/* Consultations report */
+div[data-tab="consultation_report"] {
```

Comment:
Look at the working integration, and you'll see that navigation is not with tabs but over full page reload, so this won't work. Just use a class name for the container.